### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by HoloborodkoM**

### CODE REVIEW

File: Tasks/between.js [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-      str =  str.substring(k)
-      if ( s) {
-        i = str.indexOf( s )
-        if (i === -1) {
-          return  '' } else { str =  str.substring(0 , i)
-            }}}
-  return str
-}
+const getValueBetween = (string, prefix, suffix) => {
+  const ERR = -1;
```
Comment (tshemsedinov) [inline Tasks/between.js:5]: Use `camelCase`

File: Tasks/between.js [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+  let substring = '';
+  const prefixIndex = string.indexOf(prefix);
+  const suffixIndex = string.indexOf(suffix);
+  if (prefixIndex !== ERR && suffixIndex  !== ERR) {
```
Comment (tshemsedinov) [inline Tasks/between.js:9]: Use eslint and prettier to fix code style and common conventions

File: Tasks/difference.js [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
-    delete object_1[attribute_name]
-    }{}
-    return object_1
-}
+const diff = (objectFirst, objectSecond) => {
+  const objectDiff = { ...objectFirst };
+  for (const attributeName in objectSecond) {
+    const valueObjFirst = objectFirst[attributeName];
+    const valueObjSecond = objectSecond[attributeName];
+    if (typeof(valueObjSecond) !== 'undefined') {
```
Comment (tshemsedinov) [inline Tasks/difference.js:9]: Typeof of an operator, not a function

File: Tasks/difference.js [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
-    if (object_1 in object_2) return false
-    for (attribute_name in object_2) {
-    object_1[attribute_name] = object_2[attribute_name]
-    delete object_1[attribute_name]
-    }{}
-    return object_1
-}
+const diff = (objectFirst, objectSecond) => {
+  const objectDiff = { ...objectFirst };
+  for (const attributeName in objectSecond) {
```
Comment (tshemsedinov) [inline Tasks/difference.js:6]: Prefer for..of

File: Tasks/between.js (hunk 2 above)
Comment (HoloborodkoM) [reply Tasks/between.js]: I used only eslint.

**Pull request closed without merging**